src/fabian_nemazi_ex/ex04/myrand.py

```diff
@@ -0,0 +1,39 @@
+__author__ = 'Fabian Nemazi'
+__email__ = 'fabinema@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seeds):
+        self.seeds = seeds
+
+    def rand(self):
+        a = 7**5
+        m = 2**31-1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called